クイズページ/quiz.html

```diff
@@ -50,74 +50,80 @@
         </div>
     </div>
     <main class="main">
-        <div class="quizOne">
-            <div class="quizText">
-                <h1 class="quizMidasi">
-                    Q1
-                </h1>
-                <p class="quizQuiz" id="quizQuiz">
-                    <!-- 日本のIT人材が2030年には最大どれくらい不足すると言われているでしょうか？ -->
-                </p>
-            </div>
-            <div class="quizMain">
-                <img src="" id="quizPicture">
-            </div>
-            <div class="answerIconBack">
-                <div class="answerIcon quizText">
-                    A
-                </div>
-            </div>
-            <div class="quizBlock">
-                <ul class="quizAnswer" id="quizAnswer">
-                    <!-- <li class="answerChoice">
-                        約28万人
-                        <img src="../img/icon-arrow.png" class="arrow">
-                    </li>
-                    <li class="answerChoice">
-                        <button>
+        <section id="js-content" >

```

Comment:
id名下でアッパーキャメルを使ってるからどちらかに統一したい
ex)
```suggestion
        <section id="jsContent" >

```